### PR TITLE
rocks: bump luarocks 

### LINKED ISSUE #811: Update luarocks dependency to the latest version
There were an [issue](https://github.com/luarocks/luarocks/issues/1529) in the parent luarocks/luarocks repository: `luarocks pack` would pack a rock and name it `x86_64` instead of `arm64` even if the rock itself has been built for arm.

It was fixed in 3.10 version of luarocks, but 3.11 is already available.

The related issue has been created couple months ago in https://github.com/tarantool/luarocks/issues/19.

Please update luarocks dependency so those with apple silicon chips are able to pack rocks correctly

### PR DESCRIPTION
**Pull request by better0fdead**
Updated `tarantool/luarocks` dependencie to `v3.10.0`.

Closes #811

### CODE REVIEW

**Review by oleg-jukovec — APPROVED** [review]

**Comment by oleg-jukovec** [comment]
Please, make sure that all work fine in the `tt-ee` too.

**Review by oleg-jukovec — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -30,6 +30,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 ### Changed
 
 - Do not create Dockerfile.* in application's directory.
+- Updated `tarantool/luarocks` dependency to `v3.10.0`.
```
Comment (oleg-jukovec) [inline CHANGELOG.md:33]: It will be more clear for users I think.
```suggestion
- Updated `luarocks` to `v3.10.0`.
```

**Review by DifferentialOrange — APPROVED** [review]
It is sad that luarocks repo don't have a 3.10.0 changelog, but I don't think we should do this work instead of them.

File: CHANGELOG.md (hunk 1 above)
Comment (better0fdead) [reply CHANGELOG.md]: changed

**Pull request merged**